.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+
+
+#### Brief description of what is fixed or changed
+
+<!-- If this pull request fixes an issue, write "Fixes #NNNN" in that exact
+format, e.g. "Fixes #1234". See
```

Comment:
minor: could you change it to `gh-xxxx` here and the line above? (we prefer `gh-` over `#`)

Reply:
Okay! I will do that.